src/abbreviation/abbreviations.json

```diff
@@ -909,7 +907,8 @@
     "||=": "⊫",
     "|||-": "⊪",
     "||n": "∦",
```

Comment:
I think we should remove this one and rename it to `paralleln` or `nparallel`

Reply:
`nparallel` already exists for this symbol; I removed this one.